docs/known_issues.md

```diff
@@ -6,6 +6,16 @@ This section contains current known issues and limitations with rke2. If you com
 
 Firewalld conflicts with RKE2's default Canal (Calico + Flannel) networking stack. To avoid unexpected behavior, firewalld should be disabled on systems running RKE2.
 
+## NetworkManager
+
+NetworkManager, in stock configuration, does not work well with RKE2 and Canal. As such, if installing RKE2 on a NetworkManager enabled system, it is highly recommended to configure NetworkManager to ignore calico/flannel related network interfaces. In order to do this, create a configuration file called `rke2-canal.conf` in `/etc/NetworkManager/conf.d` with the contents:
+```bash
+[keyfile]
+unmanaged-devices=interface-name:cali*;interface-name:flannel*
+```
+
+If you have not yet installed RKE2, a simple `systemctl reload NetworkManager` will suffice to install the configuration. If performing this configuration change on a system that already has RKE2 installed, a reboot of the node is necessary to effectively apply the changes.
```

Comment:
Instead of reboot of the node, can we say "restart of the rke2-canal pods"? Appears to be working for me when doing this instead of fully rebooting the nodes.

Reply:
@rancher-max its probably easier to tell people to just restart the node, vs how to find and kill the canal pods on the affected host.

Reply:
I agree with @brandond , it's a little bit of a "cannon" approach but will essentially ensure the issue will not reoccur. Otherwise, it's a multi-step reload of `NetworkManager` then restart of the `rke2-canal` pods.

Reply:
@Oats87 @brandond 
The second sentence in line 17 is not necessary. We tested and you do not need to restart the daemon or reboot the node. Only a reload is necessary. Additionally, I have a question. I thought this workaround was just for Rancher - is this not a correct assumption?
- If so, then we should clearly call out this is just for if you will be installing Rancher.
- If not, we still should mention one stipulation we noticed which is that if you are applying this workaround to run Rancher, you will need to do it first before attempting to install Rancher or Rancher will not work.

Reply:
I synced with Max and does sound like it wouldn't be a workaround for Rancher only. This is fine but can we add a mention that if you plan to install Rancher on the cluster, to do this step before doing that? This should be clearly called out to the reader.

Reply:
This is because Shylaja tested in a couple of scenarios and found that we must always ensure this is done first as a pre-req before installation of Rancher or Rancher will not work. Perhaps belongs in prereqs section.